test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs

```diff
@@ -402,6 +402,21 @@ private QueryStructuralValue BuildFromEntityInstance(EntityInstance instance, Qu
             return entity;
         }
 
+        /// <summary>
+        /// Builds an instance of QueryPrimitiveValue from an primitiveValue instance.
+        /// </summary>
+        /// <param name="value">The primitiveValue instance</param>
+        /// <param name="type">The QueryEntityType of the entity</param>
+        /// <returns>The converted structural value of entity instance.</returns>

```

Comment:
>The converted structural value of entity instance. [](start = 21, length = 50)

rename it？

Reply:
fixed

---
In reply to: [289951906](https://github.com/OData/odata.net/pull/1474#discussion_r289951906) [](ancestors = 289951906)